compose-builder/add-security.yml

```diff
@@ -233,8 +232,7 @@ services:
       KONGURL_SERVER: kong
       SECRETSERVICE_SERVER: edgex-vault
       SECRETSERVICE_TOKENPATH: /tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json
-      SECRETSERVICE_CACERTPATH: /tmp/edgex/secrets/ca/ca.pem
-      SECRETSERVICE_SNIS: "edgex-kong"
+      SECRETSERVICE_SNIS: ''
```

Comment:
Can we just remove the override? I.e. edgex-proxy should have this as blank by default if we always have it a blank?

Reply:
ok